vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -93,10 +93,23 @@ Use `ent help` to review the list of available commands
 ```
 Check a command's help text (`--help`) to view specific options and subcommands, e.g. `ent check-env --help`. For general configuration options related to ent itself, see `ent --help`.
```

Comment:
question, should we use "ent" when it's not part of a command. I feel like we should capitalize a reference to "Ent", the cli when just talking about it, but it's not a strong opinion. thoughts?

Reply:
i think we should keep it lowercase. it's either referred to as "ent" or `ent` elsewhere in the documentation. i prefer "ent" because it doesn't represent a command or entity; "ent" is the abbreviation for the Entando CLI, and a capital E looks off, as does all caps.

Reply:
Agreed, usually if you reference a command you use the exact name. Linux is case-sensitive so it definitely matters here.